GEOUNED-org/GEOUNED
# Main to Dev sync

* update Fuzzy write function

* several stp file update

* script to color CAD document following material label

* fix rounding error of z value acos(z) evaluation

* Add a bug report template and .gitignore (#44)

* Removed unused imports and variables (#35)

* raise value error instead of exit (#41)

* Update to testing path (#40)

* changed hardcoded path to work for all users

* path update

* Update bug_report.md

* raising catchable error instead of exit (#47)

* raising catchable error instead of exit

* added new line

* added try for freecad import (#48)

* Solve SyntaxtError producing a bug (#49)

* Solve SyntaxtError producing a bug

* Limit the amount of decimals printed in the enclosure comment

* fixed error units in OpenMC output

* GEOUNED training presentation added (#53)

* Adding minimal ci to check cad converts (#52)

* added minimal CI

* all step files

* added ci yml

* removed freecadcmd arg

* Update pyproject.toml

Update url links to "GEOUNED-org"

* typo correction to test if CI is working (#54)

* typo correction

* [skip ci] added ci status badge

* opening up ci to all os systems

* fix small bug with torus (bad identification of closed mozaic faces)

* Adding minimal volume checking CI comparing CAD and CSG volumes (#56)

* added volume test to ci

* removed if os cond

* added csg vs cad volume for **most** stp files

* Added mkdocs (#57)

* added example docs

* added docs deps

* Reverting mkdocs PR 57 (#58)

* Revert "added example docs"

This reverts commit bd1c67a1c786f134619e30d4a44e046eb214d8e0.

* Revert "added docs deps"

This reverts commit f31d2418c359def79be1b1fcce4bfc2053140441.

* Adding minimal particle transport tests on resulting csg files (#59)

* added transport tests

* removed files that break, raised issue

* [skip ci] remove prints

* added link to issue

* Add the name to the cells in OpenMC to improve identification (#61)

The name is the same that is used in MCNP inputs cell.Comments

* converting with no simplification = more geometry works (#62)

* reduced number of skiped geoms (#63)

* [skip ci] corrected bade to main branch (#64)

* Improved import statements (#65)

* changed single file from package to relative imports

* 2nd file converted to relative imports

* converted more fiels to relative import

* relative imports everywhere

* isort on all imports

* Add bug report and .gitignore to main (#45) (#66)



* added standard gitignore for python project (#68)

---------
